install_deps.sh

```diff
@@ -20,11 +20,11 @@ else
         fi
         cd ~/Downloads
         if [[ $(uname -m) == "x86_64" ]]; then
-            if [[ ! -d ~/Downloads/l_openvino_toolkit_p_2019.3.376 ]]; then #if the download completes and extracted, but the installation failed
-                wget http://registrationcenter-download.intel.com/akdlm/irc_nas/16057/l_openvino_toolkit_p_2019.3.376.tgz
-                tar -xvzf l_openvino_toolkit_p_2019.3.376.tgz
+            if [[ ! -d ~/Downloads/l_openvino_toolkit_p_2020.1.023 ]]; then #if the download completes and extracted, but the installation failed
+                wget http://registrationcenter-download.intel.com/akdlm/irc_nas/16345/l_openvino_toolkit_p_2020.1.023.tgz
```

Comment:
Only update to this, the raspbian runtime has also been updated: https://download.01.org/opencv/2020/openvinotoolkit/2020.1/l_openvino_toolkit_runtime_raspbian_p_2020.1.023.tgz